django/pyrog/migrations/0013_conditions_and_filters_sqlinput.py

```diff
@@ -0,0 +1,110 @@
+# Generated by Django 3.1.12 on 2021-09-09 09:28
+
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    Condition = apps.get_model("pyrog", "Condition")
+    Filter = apps.get_model("pyrog", "Filter")
+    SQLInput = apps.get_model("pyrog", "SQLInput")
+    for condition in Condition.objects.all():
+        condition.sql_input = SQLInput.objects.create(column=condition.column)
+        condition.save()
+    for filter in Filter.objects.all():
+        filter.sql_input = SQLInput.objects.create(column=filter.sql_column)
+        filter.save()
+
+
+def reverse_func(apps, schema_editor):
+    Condition = apps.get_model("pyrog", "Condition")
+    Filter = apps.get_model("pyrog", "Filter")
+    SQLInput = apps.get_model("pyrog", "SQLInput")
+
+    for condition in Condition.objects.all():
+        sql_input = SQLInput.objects.get(id=condition.sql_input.id)
+        condition.column = sql_input.column
+        condition.sql_input = None
+        sql_input.delete()
```

Comment:
This should be done automatically when columns are deleted, right?

Reply:
nope, this has to be done (otherwise the sql_input remains)